python.d/mysql.chart.py

```diff
@@ -10,9 +10,32 @@
 retries = 60
 
 # query executed on MySQL server
+QUERY_VAR = 'SHOW VARIABLES;'
 QUERY_GLOBAL = 'SHOW GLOBAL STATUS;'
 QUERY_SLAVE = 'SHOW SLAVE STATUS;'
 
+GLOBAL_VARS = [
+ 'back_log',
+ 'binlog_cache_size',
+ 'host_cache_size',
+ 'innodb_buffer_pool_size',
+ 'innodb_log_buffer_size',
+ 'innodb_log_file_size',
+ 'join_buffer_size',
+ 'key_buffer_size',
+ 'max_connections',
```

Comment:
Since you are using `max_connections` only i suggest to comment out all other list members

Reply:
That's alright a much slimmed down list of all the possible config values. I left the ones I thought others would find interesting and/or that could be added to other charts in a similar way to max_connection_limit was. Commenting them out would be a good compromise, it would make clear to others that we can fetch any config value, but (I assume) lower the memory and processing requirements for values we aren't currently using.

Reply:
Sorry for the typos there, I'm trying to reply from mobile while getting the house ready for company to come over :P

Reply:
> That's alright a much slimmed down list

I know. But there is no reason to [loop](https://github.com/firehol/netdata/pull/2642/files#diff-f580f78a2d9a19e60f501e7c13470e0eL425) through the whole list if you are not using them at the moment.